app/views/admin/daily_questions/_form.html.haml

```diff
@@ -0,0 +1,17 @@
+.field
+  %h4= f.label 'Question'
+  = f.text_field :question_id, class: 'form-control', id: 'Question', required: true
+.field
+  %h4= f.label 'Programmée le'
+  = f.text_field :scheduled_at, :id => "datepicker", value: '', class: 'form-control'
+  :javascript
```

Comment:
Like you said: this would be better to extract this into a `assets/javascripts/components/datepicker.js` file. :)